bin/exec-env

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env bash
 
-if [ "$NPM_CONFIG_PREFIX" = "" ]; then
+if [ "$NPM_CONFIG_PREFIX" = "" ] && [ "$ASDF_INSTALL_PATH" ]; then
```

Comment:
I think you need to do `[ -n "$ASDF_INSTALL_PATH" ]` here. `-n` checks that the string is not null.